E1_ShuffleNet/export.py

```diff
@@ -0,0 +1,49 @@
+import onnx
+from onnx_tf.backend import prepare
+import tensorflow as tf
+import tensorflow_hub as hub
+import os
+import io
+import requests
+from tqdm import tqdm
+import tarfile
+
+DOWNLOAD_LINK = "https://s3.amazonaws.com/download.onnx/models/opset_8/shufflenet.tar.gz"
+
+
+if not os.path.exists(DOWNLOAD_LINK.split("/")[-1]):
```

Comment:
This kind of code can make debugging tricky, since its essentially doing the job of 'main' without being included under the `main()` function. For example, in these blocks you download ShuffleNet into a protobuf file `shufflenet.pb`, which is assumed to exist in another helper.

Instead, have a function as `load_shufflenet()` that downloads (or loads) the protobuf file, and make `shufflenet.pb` a constant `SHUFFLENET_PB` thats referenced everywhere.

Reply:
Done :)